source/feature/openvswitch/__linux_native_networking_model.svg

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Adding an non-bridged VLAN with an IPconfig would be informative regarding our current networking model.

p.s. Edy, I think you've decided to use the term "legacy". if you do, please use it for the image name, too.